iphone/cli/commands/_build.js

```diff
@@ -6495,7 +6495,7 @@ iOSBuilder.prototype.optimizeFiles = function optimizeFiles(next) {
 				}
 			}
 		});
-	}(this.xcodeAppDir, /^(PlugIns|Watch|TitaniumKit\.framework)$/i));
+	}(this.xcodeAppDir, /^(PlugIns|Watch|\.framework)$/i));
```

Comment:
Do you mean `/^(PlugIns|Watch|.+\.framework)$/i`?

Reply:
Oh yeah sure, didn't notice the `^` at the beginning of the regex.